README.md

```diff
@@ -1,55 +1,82 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+The redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website.
+This includes front-end assets and build tools,
+and configuration for [Sheer](https://github.com/cfpb/sheer)
+to load content from WordPress and Django back-ends to Elasticsearch to render the site.
```

Comment:
This still feels a little confusing and doesn't include anything about the fact we use Jinja templates (which is probably important to note early). Possibly:

"This project includes the front-end assets and build tools, Jinja templates for front-end rendering, and Sheer configurations for loading content from the WordPress and Django back-ends through Elasticsearch."